src/Glpi/Console/Migration/FormCreatorPluginToCoreCommand.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2025 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Console\Migration;
+
+use Glpi\Console\AbstractCommand;
+use Glpi\Console\Traits\PluginMigrationTrait;
+use Glpi\Form\Migration\FormMigration;
+use Glpi\Progress\ConsoleProgressIndicator;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class FormCreatorPluginToCoreCommand extends AbstractCommand
+{
+    use PluginMigrationTrait;
+
+    /**
+     * Version of Formcreator plugin required for this migration.
+     * @var string
+     */
+    public const FORMCREATOR_REQUIRED_VERSION = '2.13.9';
```

Comment:
This const is only used in the `FormMigration` class, maybe it should be a private part of it instead ?